### PR TITLE
Make minor language improvement/fix punctuation

### PR DESCRIPTION
**Pull request by peterhauke**
Remove unnecessary use of the future tense

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/javascript/asynchronous/index.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ Asynchronous JavaScript is a fairly advanced topic, and you are advised to work
 ## Guides
 
 - [Introducing asynchronous JavaScript](/en-US/docs/Learn/JavaScript/Asynchronous/Introducing)
-  - : In this article we'll see what **synchronous** and **asynchronous** programming are, why we often need to use asynchronous techniques, how asynchronous functions have historically been implemented in JavaScript, and what their problems are.
+  - : In this article, we learn about **synchronous** and **asynchronous** programming, why we often need to use asynchronous techniques, how asynchronous functions have historically been implemented in JavaScript, and what their problems are.
```
Comment (dipikabh) [inline files/en-us/learn/javascript/asynchronous/index.md:41]:
```suggestion
  - : In this article, we'll learn about **synchronous** and **asynchronous** programming, why we often need to use asynchronous techniques, and the problems related to the way asynchronous functions have historically been implemented in JavaScript.
```

File: files/en-us/learn/javascript/asynchronous/index.md [hunk 2]
```diff
@@ -38,13 +38,13 @@ Asynchronous JavaScript is a fairly advanced topic, and you are advised to work
...
 - [How to use promises](/en-US/docs/Learn/JavaScript/Asynchronous/Promises)
-  - : Here we'll introduce promises and show how to use promise-based APIs. We'll also introduce the `async` and `await` keywords.
+  - : We introduce promises and show how to use promise-based APIs. We also introduce the `async` and `await` keywords.
```
Comment (dipikabh) [inline files/en-us/learn/javascript/asynchronous/index.md:43]:
```suggestion
  - : Here we'll introduce promises and show how to use promise-based APIs. We'll also introduce the `async` and `await` keywords.
```
The original text is good.

File: files/en-us/learn/javascript/asynchronous/index.md [hunk 3]
```diff
@@ -38,13 +38,13 @@ Asynchronous JavaScript is a fairly advanced topic, and you are advised to work
...
 - [Implementing a promise-based API](/en-US/docs/Learn/JavaScript/Asynchronous/Implementing_a_promise-based_API)
-  - : This article outlines how to implement your own promise-based API.
+  - : We outline how to implement your own promise-based API.
```
Comment (dipikabh) [inline files/en-us/learn/javascript/asynchronous/index.md:45]:
```suggestion
  - : This article will outline how to implement your own promise-based API.
```
Keeping future tense to be consistent with other descriptions on this page.

File: files/en-us/learn/javascript/asynchronous/index.md [hunk 4]
```diff
@@ -38,13 +38,13 @@ Asynchronous JavaScript is a fairly advanced topic, and you are advised to work
...
 - [Introducing workers](/en-US/docs/Learn/JavaScript/Asynchronous/Introducing_workers)
-  - : Workers enable you to run certain tasks in a separate thread, to keep your main code responsive. In this article we'll rewrite a long-running synchronous function to use a worker.
+  - : Workers enable you to run certain tasks in a separate thread to keep your main code responsive. In this article, we rewrite a long-running synchronous function to use a worker.
```
Comment (dipikabh) [inline files/en-us/learn/javascript/asynchronous/index.md:47]:
```suggestion
  - : Workers enable you to run certain tasks in a separate thread to keep your main code responsive. In this article, we'll rewrite a long-running synchronous function to use a worker.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for your edits here, @peterhauke! Some of my responses here are similar to the ones in https://github.com/mdn/content/pull/17389. Let me know what you think.

File: files/en-us/learn/javascript/asynchronous/index.md (hunk 1 above)
Comment (peterhauke) [reply files/en-us/learn/javascript/asynchronous/index.md:41]: Yes, much easier to understand.

**Comment by peterhauke** [comment]
Thanks, @dipikabh

Yes, I thought this was going to happen to my edits here!

**Review by dipikabh — APPROVED** [review]
Thanks for your work on this PR, @peterhauke!

**Pull request merged**